Assets/Scripts/Actor/Living.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Actor {
+    /// <summary>
+    /// Function that keeps track of health and 
+    /// any resources dealing with morbid death.
+    /// </summary>
+    public class Living : SlipTime.SlipTimeAdherent
+    {
+        public Color damageTint;
+        // Private Variables
+        [SerializeField]
+        private int health = 100;
+
+        [SerializeField]
+        private Animator animator;
+
+        [SerializeField]
+        private float colorDecayTime = 2.0f;
+        private SpriteRenderer spriteRenderer;
+        private bool takenDamage = false;
+        private float timeCounter = 0;
+
+        // Getters and Setters
+        public int Health => this.health;
+        public Color DamageTint => this.damageTint;
+        public float ColorDecayTime => this.colorDecayTime;
+
+        void Start()
+        {
+            animator = gameObject.GetComponent<Animator>();
+            spriteRenderer = gameObject.GetComponent<SpriteRenderer>();
+        }
+
+        void Update()
+        {
+            if (health <= 0)
+            {
+                Die();
+            }
+
+            // Basic damage effect via scripts
+            // we can decide to use sprites instead later down the line for crazier effects
+            if (takenDamage)
+            {
+                Color lerpedColor = Color.Lerp(damageTint, Color.white,  timeCounter);
```

Comment:
nit: I think there's an extra space between the comma after `Color.white` and `timeCounter`.